Configuration/Geometry/python/dict2021Geometry.py

```diff
@@ -39,6 +67,208 @@
     "name" : "tracker",
     "default" : 1,
     "T1" : {
+        1 : [
```

Comment:
I think this is a mistake: T1 is changed to have the zero material xml files, whereas those should be placed in T2

Reply:
Should I swap T1 and T2?  What about O1 and O2 - I put O2 as zero material one. How to run the script then - could you modify the script and let me know how to use it? Also README.md should be significantly changed